aerogear/mobile-security-service-operator
# feat(mss monitoring alerts): Add alerts for monitoring http, cpu, mem and uptime (INTLY-1893)

## Motivation
https://issues.jboss.org/browse/INTLY-1893

## What
Add alerts for the MSS service

## Why
alerts for monitoring http, cpu, mem and uptime

## How
Added alerts to prometheus for middleware monitoring stack

## Verification Steps
Not easy to verify, the alerts limits are hard to reproduce
Check the alerts are present at 
https://prometheus-route-middleware-monitoring.apps.austin-81cc.openshiftworkshop.com/alerts

These should be available
![image](https://user-images.githubusercontent.com/16667688/59929860-97108600-9439-11e9-8c94-906d83ad6314.png)

- check the **MobileSecurityServiceDown** **MobileSecurityServiceDatabaseDown** by scaling down the operator and the mobile-security-service and the mobile-security-service-db for 5 min to see if alert triggers


## Checklist:
